src/components/UI/AttachmentLink/AttachmentLink.tsx

```diff
@@ -24,6 +24,43 @@ interface OriginalDocumentProps extends v2.OpenAttestationDocument {
   };
 }
 
+interface ExtensionIconProps {
+  src: string;
+}
+
+const ExtensionIcon: FunctionComponent<ExtensionIconProps> = ({ src }: ExtensionIconProps) => {
+  return (
+    <img
+      src={`/static/images/fileicons/${src}.svg`}
+      className="flex items-center justify-center mr-2"
+      data-testid={`attachment-icon-${src}`}
```

Comment:
mmm why add `data-testid`? r u going to write a test? maybe u should do 1 unit test for the `getExtension` part